core/src/replay_stage.rs

```diff
@@ -2896,6 +2897,34 @@ impl ReplayStage {
                     (bank.slot(), bank.hash()),
                     Some((bank.parent_slot(), bank.parent_hash())),
                 );
+                // If the block does not have at least 64 shreds in the last FEC set, mark
+                // it as duplicate, effectively removing it from fork choice.
+                let mut incomplete_last_fec_set = true;
+                let slot_meta = blockstore
+                    .meta(bank.slot())
+                    .expect("Slot meta get must succeed on frozen banks")
+                    .expect("Slot meta must exist during freeze");
+                if let Some(last_shred_index) = slot_meta.last_index {
+                    if let Ok(Some(erasure_meta)) = blockstore.erasure_meta(ErasureSetId::new(
+                        bank.slot(),
+                        u32::try_from(last_shred_index).expect("LAST_SHRED_IN_SLOT should be u32"),
+                    )) {
+                        if erasure_meta.total_shreds() >= 64 {
```

Comment:
We should probably get this constant from somewhere else depending on how the other PR is implemented.